### PR TITLE
feature 5 complete - user can comment

### PR DESCRIPTION
**Pull request by bryanvaldovinos**
ezgif com-gif-maker

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -56,77 +56,195 @@ xhr.addEventListener('load', function parkList() {
...
+      var propArray = Object.keys(data.comment);
+      for (var q = 0; q < propArray.length; q++) {
+        if ($comment[w].dataset.dig === propArray[q]) {
+          $comment[w].elements[0].textContent = data.comment[w];
+          data.comment[w] = $comment[w].elements[0].value;
+          $comment[w].elements[1].className = 'hidden';
+          $comment[w].elements[0].className = 'text-marg border-r5 green-bg';
+        }
+      }
+    }
```
Comment (codingcodymiller) [inline js/main.js:136]: This would be a good thing to make into its own named function.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by bryanvaldovinos** [comment]
https://user-images.githubusercontent.com/103273265/175613579-0af7d74e-b2b8-4927-8c21-f2d3383df887.mov

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
I didn't mean to make the conditional part of the function definition too, your conditional could stay as it was, just the contents of each conditional could be its own function.

**Comment by bryanvaldovinos** [comment]
ezgif com-gif-maker (1)

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**